frontend/src/actions/homeActions.js

```diff
@@ -265,6 +271,7 @@ export const buildFilterData = () => async (dispatch, getState) => {
 };
 
 export const fetchDataConcurrently = () => async (dispatch) => {
+  console.log("why doing again");
```

Comment:
Why *not* do it again, if it had so much fun the first time?

(Or ... this undoubtedly made sense for debugging, but should probably be removed now!)